### DISCUSSION TITLE
✉ Where can I contact you ? ✉

### DISCUSSION THREAD

**Discussion by Naaikho** [demberto/PyFLP#73] [General]
Hi, I'm currently working on a project (application) and I'm using your PyFLP on a Python server using SocketIO to exchange data.
I wanted to contact you in some way to talk about it and find out if it was possible to add some rather useful things to the library.

for information, I have [my social networks](https://naikho.com) on my profile so if you prefer not to disclose anything here you can always write to me directly. 👌

Naikho

**Comment by demberto** [comment]
You can contact me on my email demberto@proton.me or my Discord Dem Burto#9216.

> I wanted to contact you in some way to talk about it and find out if it was possible to add some rather useful things to the library.

You can open a new issue for a feature request as well, if its convenient for you :)
